Makefile

```diff
@@ -24,13 +24,21 @@ ifneq ($(MAKECMDGOALS),clean)
     ifeq (0,$(shell which pyside-rcc >/dev/null 2>&1; echo $$?))
       RCC = pyside-rcc
     else
-      $(error "resource compiler not found")
+      ifeq (0,$(shell which pyside2-rcc >/dev/null 2>&1; echo $$?))
```

Comment:
Is there a particular reason for checking for PySide2's resource compiler /after/ PySide's? Would have thought you'd want this the other way around. Additionally, the Python changes drop support for PySide, so what point is there keeping the old build logic around?